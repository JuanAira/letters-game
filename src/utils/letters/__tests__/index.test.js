import { generateBoard } from "..";

describe('generateBoard', () => {
  it('should match snapshot - random', () => {
    const container = generateBoard({ defaultNumber: true });
    expect(container).toMatchSnapshot();
  });
});
