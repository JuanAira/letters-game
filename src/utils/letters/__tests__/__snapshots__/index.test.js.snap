// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateBoard should match snapshot - random 1`] = `
Object {
  "board": Array [
    "A",
    "B",
    "C",
    "D",
    "E",
    "F",
    "G",
    "H",
    "I",
    "J",
    "K",
    "L",
    "M",
    "N",
    "O",
    "P",
  ],
  "neighbors": Object {
    "0": Array [
      4,
      5,
      1,
    ],
    "1": Array [
      0,
      4,
      5,
      6,
      2,
    ],
    "10": Array [
      5,
      9,
      13,
      6,
      14,
      7,
      11,
      15,
    ],
    "11": Array [
      7,
      6,
      10,
      14,
      15,
    ],
    "12": Array [
      8,
      9,
      13,
    ],
    "13": Array [
      14,
      10,
      9,
      8,
      12,
    ],
    "14": Array [
      13,
      9,
      10,
      11,
      15,
    ],
    "15": Array [
      14,
      10,
      11,
    ],
    "2": Array [
      1,
      5,
      6,
      7,
      3,
    ],
    "3": Array [
      2,
      6,
      7,
    ],
    "4": Array [
      0,
      1,
      5,
      9,
      8,
    ],
    "5": Array [
      0,
      1,
      2,
      6,
      10,
      9,
      8,
      4,
    ],
    "6": Array [
      1,
      5,
      9,
      2,
      10,
      3,
      7,
      11,
    ],
    "7": Array [
      3,
      2,
      6,
      10,
      11,
    ],
    "8": Array [
      4,
      5,
      9,
      13,
      12,
    ],
    "9": Array [
      4,
      8,
      12,
      5,
      13,
      6,
      10,
      14,
    ],
  },
}
`;
